swan-lake/learn-the-platform/build-a-graalvm-executable/build-the-executable-locally.md

```diff
@@ -10,25 +10,23 @@ intro: This guide walks you through compiling a Ballerina application to a nativ
 ## Prerequisites
 
 To complete this part of the guide, you need:
-1. [Ballerina 2201.7.0 (Swan Lake)](/downloads) or greater
+1. [Latest Ballerina Swan Lake distribution](/downloads)
 2. A text editor
    >**Tip:** Preferably, <a href="https://code.visualstudio.com/" target="_blank">Visual Studio Code</a> with the  <a href="https://wso2.com/ballerina/vscode/docs/" target="_blank">Ballerina extension</a> installed.
 3. GraalVM installed and configured appropriately
 4. A command terminal
 
 ## Configure GraalVM
 
-1. Install GraalVM on Linux, macOS, and Windows (via Git Bash, Cygwin, or WSL) using the following command.
+1. Install GraalVM using [SDKMAN!](https://sdkman.io/). For additional download options, see [Get Started with GraalVM](https://www.graalvm.org/latest/docs/getting-started/).
```

Comment:
```suggestion
1. Install GraalVM using [SDKMAN!](https://sdkman.io/). For additional download options, see [Get Started with GraalVM](https://www.graalvm.org/latest/docs/getting-started/).
```
Make the second sentence a tip.

Reply:
Fixed via [2d302dd](https://github.com/ballerina-platform/ballerina-dev-website/pull/7676/commits/2d302ddb7f82fd79ff8579a97fd7140604c41df3)